code/game/gamemodes/gang/dominator.dm

```diff
@@ -13,6 +13,7 @@
 	var/datum/gang/gang
 	var/operating = 0	//0=standby or broken, 1=takeover
 	var/warned = 0	//if this device has set off the warning at <3 minutes yet
+	var/spam_prevention = 100
```

Comment:
Just a way to make sure it sends a message immediately when it's blocked (which then resets prevention to 0). So if I adjust the prevention window down the road - this won't have to be altered.